picky-krb/src/crypto/aes/aes256_cts_hmac_sha1_96.rs

```diff
@@ -36,13 +36,9 @@ impl Cipher for Aes256CtsHmacSha196 {
     }
 
     fn encrypt(&self, key: &[u8], key_usage: i32, payload: &[u8]) -> Result<Vec<u8>, KerberosCryptoError> {
-        encrypt_message(
-            key,
-            key_usage,
-            payload,
-            &AesSize::Aes256,
-            OsRng.r#gen::<[u8; AES_BLOCK_SIZE]>(),
-        )
+        let mut confounder = [0; AES_BLOCK_SIZE];
+        OsRng.try_fill_bytes(&mut confounder)?;
+        encrypt_message(key, key_usage, payload, &AesSize::Aes256, confounder)
```

Comment:
question: Should we consider StdRng which implements the non-try version?

Reply:
Done